torch/_refs/__init__.py

```diff
@@ -771,6 +793,22 @@ def _logical_or(a: TensorLikeType, b: TensorLikeType):
     aten_op=torch.ops.aten.logical_or,
 )
 
+
+def _logical_xor(a: TensorLikeType, b: TensorLikeType):
```

Comment:
It's funny that since _logical_xor is (correctly) modeled as an elementwise binary operator it will perform type promotion by default. That is, if it's given a long tensor and a float tensor then the long tensor will be first be cast to float before both will be cast to boolean.

The unnecessary conversion of long to float can be removed when analyzing the trace, but we might want to add a TODO comment here noting that a more efficient implementation would avoid the `_make_elementwise_binary_reference` helper to skip type promotion